db/models.py

```diff
@@ -1 +1,32 @@
 from django.db import models
+
+
+class Race(models.Model):
+    RASE = [
+        ("E", "elf"),
+        ("D", "dwarf"),
+        ("H", "human"),
+        ("O", "ork")
+    ]
```

Comment:
The variable `RASE` should be renamed to `RACES` to better reflect its purpose as a collection of race choices. This will improve code readability and maintain consistency with common naming conventions.